docs/guides/cli.md

```diff
@@ -61,6 +62,65 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request a specific example from a resource
+
+You can request a specific example from your document by using the `__example` query string paramenter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__example=exampleKey
+```
+
+#### Change the dynamic/static flag for a particular request
+
+You can override the `dynamic` keyword (which decides whether the generated example is static or dynamic) through the `__dynamic` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
+```
+
+## Proxy
+
+This command creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
+
+**Note:** Currently the proxy command _expects_ a JSON Payload for both the request and the response. If the response's `content-type` header does not indicate a JSON-ish payload (`application/json`, `application/*+json`), then the payload will be just returned as text and send back to the original client.
+
+```
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2
+
+[CLI] …  awaiting  Starting Prism…
+[HTTP SERVER] ℹ  info      Server listening at http://127.0.0.1:4010
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets
+[CLI] ●  note      POST       http://127.0.0.1:4010/pets
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets/10
+```
+
+By default, the output violations will be reported on the standard output, but you can also specify to embed these in the HTTP Response:
+
+```bash
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2 --log httpHeader
+… …
+
+curl -v -s http://localhost:4010/pets/10 > /dev/null
+
+< HTTP/1.1 404 Not Found
+< sl-validation: [{"severity":"Error","message":"The received media type does not match the one specified in the document"},{"location":["body"],"severity":"Error","code":"type","message":"should be object"}]
```

Comment:
I'm looking into the `warning` header as a more standard alternative to this. https://tools.ietf.org/html/rfc7234#section-5.5

Reply:
I had a quick look into and it seems to be mostly cache related?

Reply:
2nd paragraph says: 

>    Warnings can be used for other purposes, both cache-related and otherwise.  The use of a warning, rather than an error status code, distinguishes these responses from true failures.

Reply:
I'm sorry I totally missed that part. Seems to fit very well with what we're looking for. Do you want me to make the change?

Reply:
Yeah let’s do it. It might be tricky to fit the information into text but if people want json they can enable the —error mode or whatever.

Reply:
@philsturgeon So if I understand correctly in the `warning` header we're just going to put the error message and no other informations.